flakytests/cyclers/30-r1_at_icp_or

```diff
@@ -1 +0,0 @@
-19-async_integer
```

Comment:
What's the logic behind converting this symlink to a copy of the linked file?

This symlinking approach was purposeful and is used in quite a few tests. It reduces duplication and makes the test battery faster to fix.

Reply:
I didn't realise - I've rebased to keep the symlink `30-r1_at_icp_or.t` and only removed the incorrect folder symlink `30-r1_at_icp_or/`

Reply:
No probs, it't not a pattern I'm especially fond of!